### DISCUSSION TITLE
Unable to install SQL Server on Remote VM - All Firewalls disabled.

### DISCUSSION THREAD

**Discussion by anilmahadev** [dataplat/dbatools#9191] [Q&A] [answered]
Hi All,

I am running into the same issue.

I have disabled Firewalls between my AdminDbatools Win11 VM and my Windows Server 2022 as well.

==

Install-Dbainstance -Version 2022 -SqlInstance VM01 -Feature Engine -Path C:\SQL2022 -AuthenticationMode Mixed

WARNING: [23:47:12][Install-DbaInstance] Failed to get reboot status from VM01| Multiple protocol connections were attempted, but no successful connections could be established with the specified computer.

Able to ping host to VM VM and to host as well.

The version installed is :

PS C:\> Get-InstalledModule dbatools

Version              Name                                Repository           Description
-------              ----                                ----------           -----------
2.1.6                dbatools                            PSGallery            The community module that enables SQL Se…

==

PS C:\> Install-Dbainstance -Version 2022 -SqlInstance VM01  -Feature Engine -Path C:\SQL2022 -AuthenticationMode Mixed
WARNING: [01:25:17][Install-DbaInstance] Failed to get reboot status from teamservervm | Windows authentication was used, but failed

I don't understand where I am going wrong.

Appreciate your help in this regard.

Cheers!

**Comment by anilmahadev** [comment]
Here is a screenshot from the VM where I have placed the SQL2022 installer files extracted. the powershell window is from my local laptop. 

![image](https://github.com/dataplat/dbatools/assets/6380886/0d0d0d95-c618-4405-bef7-857d85e4599a)

Thanks
Please let me know if you have any questions or things I need to try again

**Comment by anilmahadev** [comment]
Hi All,
I will move on to the next set of steps of learning this book, as I don't wish to be get buried in the weeds.. 

I will setup a DC and add a SQL2019 and SQL2022. 

Try then.

Problem 1: I however I got over the initial huddle of not being able to talk to the Remote machine.. 

Solution 1: Used winrm quickconfig on the remote machine. rebooted.

problem 2: 

WARNING: [20:44:20][Install-DbaInstance] [SQL2022DT] Connecting to remote server SQL2022DT failed with the following error message : The WinRM client cannot process the request. A computer policy does not allow the delegation of the user credentials to the target computer because the computer is not trusted. The identity of the target computer can be verified if you configure the WSMAN service to use a valid certificate using the following command: winrm set winrm/config/service '@{CertificateThumbprint="<thumbprint>"}'  Or you can check the Event Viewer for an event that specifies that the following SPN could not be created: WSMAN/<computerFQDN>. If you find this event, you can manually create the SPN using setspn.exe .  If the SPN exists, but CredSSP cannot use Kerberos to validate the identity of the target computer and you still want to allow the delegation of the user credentials to the target computer, use gpedit.msc and look at the following policy: Computer Configuration -> Administrative Templates -> System -> Credentials Delegation -> Allow Fresh Credentials with NTLM-only Server Authentication.  Verify that it is enabled and configured with an SPN appropriate for the target computer. For example, for a target computer name "myserver.domain.com", the SPN can be one of the following: WSMAN/myserver.domain.com or WSMAN/*.domain.com. Try the request again after these changes. For more information, see the about_Remote_Troubleshooting Help topic.

Solution: I suppose I need to add the remote machine as a trusted source.

I will keep you all posted.

Thanks

**Comment by anilmahadev** [comment]
I would like to report that I was encountering issues with Windows 11 and Windows Server non-domain joined.

I will go ahead and close this and try out the Workgoup related instances in another lab. But for now, I have everything I need to progress in the book :)

Cheers!
![Screenshot 2023-12-27 000037](https://github.com/dataplat/dbatools/assets/6380886/87b10ffd-f068-4f83-aafb-1bbe032f55b7)

**Reply by wsmelton** [reply, marked as answer]
Working in a workgroup the account you are using has to exist with the exact same password between your devices. If you login to Windows11 as Administrator with something like `p@ssword123` then that account on Server1 must have the same password.

**Reply by anilmahadev** [reply]
Thank you Sean for clearing that up. Will give it another whirl with your suggestion :). Cheers
Anil :)
